### DISCUSSION TITLE
Details about the Fixed Version runtime distribution mode

### DISCUSSION THREAD

**Discussion by captainbrosset** [MicrosoftEdge/WebView2Feedback#3926] [Q&A]
Posted by @JaseRandall in https://github.com/MicrosoftDocs/edge-developer/issues/1413

> I've followed the Details about the Fixed Version runtime distribution mode instructions and get an error: Microsoft.Web.WebView2.Core.WebView2RuntimeNotFoundException: Couldn't find a compatible Webview2 Runtime installation to host WebViews. ---> System.IO.FileNotFoundException: The system cannot find the file specified. (Exception from HRESULT: 0x80070002)
> 
> Can you please update the instructions to include a code example. I need to deploy the WebView2 binaries onto a system for which I don't have administrator privileges so I don't have the option to use the non Fixed Version runtime approaches.

This was related to this documentation page: https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution

Posting here as this issue requires someone with WebView2 technical knowledge to help if possible. Thanks!

**Comment by champnic** [comment]
Hey @JaseRandall!

Can you confirm that the Fixed Version binaries are where you expect them to be on the machine?

Have you followed these steps, specifically Step 4, to point the WebView2 to use the Fixed Version binaries?

https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#details-about-the-fixed-version-runtime-distribution-mode

Thanks!

**Comment by Ryanma-YX** [comment]
> 

hey,
I had the same problem with the fixed version,
Webview2 works fine in Winform (.net 4.6.1),
In WPF (.net 5), the same code,the exception thrown is WebView2RuntimeNotFoundException

```cs
 var options = new Microsoft.Web.WebView2.Core.CoreWebView2EnvironmentOptions("--disable-web-security");   

            var browserFolder = Path.Combine(RootDir,BrowserFolder); //C:\Users\Administrator\AppData\Local\ERPBrowser\Plugin
            var userDataFolder = Path.Combine(RootDir, UserDataFolder); //C:\Users\Administrator\AppData\Local\ERPBrowser\Cache
            if (!Directory.Exists(userDataFolder))
            {
                Directory.CreateDirectory(userDataFolder);
            }
            var env = await Microsoft.Web.WebView2.Core.CoreWebView2Environment.CreateAsync(browserFolder, userDataFolder, options); // exception  here
            await WebViewBrowser.EnsureCoreWebView2Async(env);
```

**Comment by champnic** [comment]
Hey @ryanryanorient - Can you confirm that the browserFolder path is pointing to a directory on your machine that has the FixedRuntime in it? If you put the fixed runtime in a different directory does that work? Is "RootDir" the exact same in both the Winforms and WPF cases?
